.gitignore

```diff
@@ -85,3 +85,6 @@ venv.bak/
 
```

Comment:
Forgot to mention in description that I updated HttpCaller to support dynamically-provided basic & bearer auth.